oemof/solph/inputlib/csv_tools.py

```diff
@@ -11,7 +11,7 @@
 
 def NodesFromCSV(file_nodes_flows, file_nodes_flows_sequences,
                  delimiter=',', additional_classes={},
-                 additional_seq_attributes=[], addtional_flow_attributes=[]):
+                 additional_seq_attributes=[], additional_flow_attributes=[]):
```

Comment:
> It does not really belong to you changes, so you may ignore this comment but your default attributes are mutable.
> 
> It might not be a problem in our case but I would like to be in line with common python rules.
> 
> http://docs.python-guide.org/en/latest/writing/gotchas/

This has been added from @simnh if I remember it right and I think he just needed the functionality quickly and didn't think about it too much. Neverthess, you are right!

Should we fix this?